core/src/test/java/com/linecorp/armeria/client/RefreshingAddressResolverTest.java

```diff
@@ -405,6 +406,7 @@ private static DnsResolverGroupBuilder builder(TestDnsServer... servers) {
                         Stream.of(servers).map(TestDnsServer::addr).collect(toImmutableList())).stream();
         return new DnsResolverGroupBuilder()
                 .dnsServerAddressStreamProvider(dnsServerAddressStreamProvider)
+                .meterRegistry(PrometheusMeterRegistries.newRegistry())
```

Comment:
Could you explain why we have this change? This test case doesn't seem to check and DNS metrics.

Reply:
Good question @trustin. The reason is DnsResolverGroupBuilder::build method we expect the meterRegistry to be not null. The question can DnsResolverGroupBuilder be constructed without meterRegistry.

Reply:
Ahh, I see. OK!